nucliadb_utils/nucliadb_utils/grpc.py

```diff
@@ -78,7 +80,10 @@ def get_traced_grpc_server(service_name: str, max_receive_message: int = 100):
     tracer_provider = get_telemetry(service_name)
     if tracer_provider is not None:  # pragma: no cover
         otgrpc = GRPCTelemetry(f"{service_name}_grpc", tracer_provider)
-        server = otgrpc.init_server(max_receive_message=max_receive_message)
+        server = otgrpc.init_server(
+            max_receive_message=max_receive_message,
+            interceptors=[SentryInterceptor()],
```

Comment:
This should only be conditional to sentry being configured right?

Reply:
I'll change it to use `nucliadb_telemetry.errors.capture_exception`, that already handles this